assets/js/sse.js

```diff
@@ -91,15 +92,15 @@ var SSE = function (url, options) {
   this._onStreamFailure = function(e) {
     this.dispatchEvent(new CustomEvent('error'));
     this.close();
-  }
+  };
```

Comment:
OK, but I'd go further. SSE.js is polyfill, used **exclusively** to show popups to user using [Notification API](https://developer.mozilla.org/en-US/docs/Web/API/notification) which is also [unsupported on IE11](https://caniuse.com/notifications). I suggest removing this library.

Reply:
If we remove this library, it means removing the notifications feature entirely, which is not wanted.
We can instead use this library only if the Notification API is supported by the browser.

Reply:
Oh, I've not expressed accurately. I'm dont want to cut feature, I want to cut useless in this case code. Let me explain little more.
This polyfill is useless to IE11, as it doesn't support [Notification API](https://caniuse.com/notifications) and useless for browsers that support this API, because [all of them have support of Server-sent events](https://caniuse.com/eventsource) with native `EventSource` class. So we may replace this library to `EventSource` without harm.

Reply:
Oooh, okay! Thanks for explaining! In that case, yes, feel free to propose an alternative to SSE.js in another PR ^^